lib/node-graceful.ts

```diff
@@ -0,0 +1,167 @@
+// +----------------------------------------------------------------------+
+// | node-graceful v3 (https://github.com/mrbar42/node-graceful)      |
+// | Graceful process exit manager.                                       |
+// |----------------------------------------------------------------------|
```

Comment:
Inlined fork of node-graceful because the module on NPM is broken when running ESM.